src/modules/reporting/components/reporting-dispute/reporting-dispute.jsx

```diff
@@ -65,6 +68,25 @@ export default class ReportingDispute extends Component {
             message="There are currently no markets available for dispute."
           />
         }
+        <article className={MarketsHeaderStyles.MarketsHeader}>
+          <h4 className={MarketsHeaderStyles.MarketsHeader__subheading}>Upcoming Dispute Window</h4>
+        </article>
+        {p.upcomingMarketsCount > 0 &&
+            p.upcomingMarkets.map(market =>
+              (<DisputeMarketCard
+                key={market.id}
+                market={market}
+                disputeRound={p.disputeRound}
+                isMobile={p.isMobile}
+                location={p.location}
+                history={p.history}
+              />))
+        }
+        { p.upcomingMarketsCount === 0 &&
+          <NullStateMessage
+            message="No markets in this dispute window have been successfully disputed."
```

Comment:
Is this copy from the ticket? Was it discussed elsewhere?

Reply:
Ah I got mine off of the mock screenshot, will change